tests/test_logs_db.py

```diff
@@ -0,0 +1,84 @@
+import asyncpg
+import pytest
+
+from autoagora.logs_db import LogsDB
+
+
+class TestLogsDB:
+    @pytest.fixture
+    async def pgpool(self, postgresql):
+        pool = await asyncpg.create_pool(
+            host=postgresql.info.host,
+            database=postgresql.info.dbname,
+            user=postgresql.info.user,
+            password=postgresql.info.password,
+            port=postgresql.info.port,
+        )
+        assert pool
+        await pool.execute(
+            """
+            CREATE TABLE query_skeletons (
+                hash BYTEA PRIMARY KEY,
+                query TEXT NOT NULL
+            )
+        """
+        )
+        await pool.execute(
+            """
+            CREATE TABLE query_logs (
+                id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+                query_hash BYTEA REFERENCES query_skeletons(hash),
+                subgraph CHAR(46) NOT NULL,
+                timestamp TIMESTAMPTZ NOT NULL,
+                query_time_ms INTEGER,
+                query_variables TEXT
+
+            )
+        """
+        )
+        await pool.execute(
+            """
+            INSERT INTO query_skeletons (hash, query)
+            VALUES ('hash1', 'query getData{ values { id } }'), ('hash2', 'query getInfo{ info { id text} }')
+        """
+        )
+        await pool.execute(
+            """
+            INSERT INTO query_logs (query_hash, subgraph, timestamp, query_time_ms)
+            VALUES ('hash1', 'GoRT5CG6EBfBmLjwkB8sRDl5xh2IllCZneaah4KE2eRw3K', '2023-05-18T21:47:41+00:00', 100),
+            ('hash1', 'GoRT5CG6EBfBmLjwkB8sRDl5xh2IllCZneaah4KE2eRw3K', '2023-05-18T21:47:41+00:00', 200),
+            ('hash2', 'Z6wugcyUmABUzx3jDEevc84RvfIpVGKNpciEXmci3eT9s2', '2023-05-18T21:47:41+00:00', 50),
+            ('hash1', 'Z6wugcyUmABUzx3jDEevc84RvfIpVGKNpciEXmci3eT9s2', '2023-05-18T21:47:41+00:00', 10)
```

Comment:
These are not valid IPFS hashes. That could be a problem is we ever introduce some kind of validation somewhere. See https://ethereum.stackexchange.com/a/53233